monitoring.adoc

```diff
@@ -787,22 +787,31 @@ This pattern consists of 6 blocks of data:
 
 * Date and time (e.g.: `2018-01-22 19:23:07,100`)
 * Log level (e.g. `INFO`)
-* Thread number with a prefix (e.g. `[qtp1043351526-547:https:https://localhost:7071/service/admin/soap/DeleteAccountRequest]`, `[Index-9]`, etc)
+* Thread name (e.g. `[qtp1043351526-547:https:https://localhost:7071/service/admin/soap/DeleteAccountRequest]`, `[Index-9]`, etc)
 * {product-name} context
 * Component name (e.g. `soap`, `mailbox`, `mbxmgr`, etc)
 * Log message. *Note:* the log message section may span multiple lines. When a log message contains an exception, the stack trace will always start on a new line below the error message.
```

Comment:
Not sure if its possible but rather than the second sentence "When a log messsage contains ..." line can we do something like "See "Handler Exceptions and Stack Traces" for more details" and have the title be a link to that specific section.